### DISCUSSION TITLE
Testers Needed!

### DISCUSSION THREAD

**Discussion by rudybrian** [rudybrian/PyNavienSmartControl#1] [Announcements]
## 👋 Testers Needed!
  We can use some help testing various configurations. Specifically testing reports from users with the following configurations would be extremely helpful:
  * Units equipped with outdoor temperature sensors.
  * Combi-boiler (DHW and CH) configurations
  * Cascaded units
  * CH-only boilers
  * Units equipped with a Hot Button
  * Non-US units using metric

**Comment by tc60045** [comment]
I have 2 Navilinks
one to a pair of cascading NFB-200s
one to a single NFB-200 (guest house)

I am new to the home-assistant.io group as I've managed lots of different smart apps without needing integration, but now want a bit more integration.   

I develop in python, contribute to a few FOSS groups - happy to help you test.
